NumericOutcome.md

```diff
@@ -0,0 +1,453 @@
+`m` is any positive number less than or equal to `n`, the sum of the corresponding signature
+points is used: `s(1..m) * G = (s1 + s2 + ... + sm) * G = s1 * G + s2 * G + ... + sm * G`.
+
+When the oracle broadcasts its `n` signatures `s1, ..., sn`, the corresponding adaptor secret can be
+computed as `s(1..m) = s1 + s2 + ... + sm` which can be used to broadcast the CET.
+
+#### Rationale
+
+This design allows implementations to re-use all [transaction construction code](Transactions.md) without modification
+because every CET needs as input exactly one adaptor point just like in the single-nonce setting.
+
+Another design that was considered was adding keys to the funding output so that parties could collaboratively
+construct `m` adaptor signatures and where `n` signatures are put on-chain in every CET which would reveal
+all oracle signatures to both parties when a CET is published.
+This design's major drawbacks is that it creates a very distinct fingerprint and makes CET fees significantly worse.
+This design's only benefit is that it results in simpler (although larger) fraud proofs.
+
+The large multi-signature design was abandoned because the above proposal is sufficient to generate fraud proofs.
+If an oracle incorrectly signs for an event, then only the sum of the digit signatures `s(1..m)`
+is recoverable on-chain using the adaptor signature which was given to one's counter-party.
+This sum is sufficient information to determine what was signed however as one can iterate through
+all possible composite adaptor points until they find one whose pre-image is the signature sum found on-chain.
+This will determine what digits `(d1, ..., dm)` were signed and these values along with the oracle
+announcement and `s(1..m)` is sufficient information to generate a fraud proof in the multi-nonce setting. 
+
+## Contract Execution Transaction Compression
+
+Anytime there is a range of numeric outcomes `[start, end]` which result in the same payouts for all parties,
+then a compression function can be run to reduce the number of CETs to be `O(log(L))` instead of `L` where 
+`L = end - start + 1` is the length of the interval being compressed.
+
+Most contracts are expected to be concerned with some subset of the total possible domain and every
+outcome before or after that range will result in some constant maximal or minimal payout.
+This means that compression will drastically reduce the number of CETs to be of the order of the size
+of the probable domain, with further optimizations available when parties are willing to do some rounding.
+
+The compression algorithm takes as input a range `[start, end]`, a base `B`, and the number of digits
+`n` (being signed by the oracle) and returns an array of arrays of integers (which will all be in the range `[0, B-1]`).
+An array of integers corresponds to a single event equal to the concatenation of these integers (interpreted in base `B`).
+
+### Example
```

Comment:
Your example is too complex and you add too much details (which should be included somewhere though). Better to take a simple example in base 10 so that it reads easily, and then convince that actually base 2 is better.